Makefile

```diff
@@ -19,10 +19,10 @@
 
 # values used in workspace-status.sh
 
-DOCKER_REGISTRY?=cockroachdb
+DOCKER_REGISTRY?=us.gcr.io/chris-love-operator-playground
 DOCKER_IMAGE_REPOSITORY?=cockroachdb-operator
 # Default bundle image tag
-APP_VERSION?=v1.6.12-rc.2
+APP_VERSION?=v1.8.41
```

Comment:
@keith-mcclellan we need to start thinking about versioning

Reply:
Agree - let's discuss tomorrow after the demo